terraform/concourse/concourse.tf

```diff
@@ -159,7 +159,9 @@ resource "terraform_data" "provision_concourse" {
     command = <<-CMD
       gcloud compute ssh ${var.hostname} --zone ${var.zone} --tunnel-through-iap --command "\
       sudo /concourse/scripts/install.sh ${var.docker_ver} ${var.data_disk} ;\
-      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert}" --verbosity=error
+      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert} ;\
+      // cron tab runs every weekend e.g. on Sunday at midnight ;\
+      ! grep -q '0 0 * * 0 /concourse/scripts/prune_workers.sh' && echo '0 0 * * 0 /concourse/scripts/prune_workers.sh' | sudo tee -a /etc/crontab" --verbosity=error
```

Comment:
please note that ubuntu / debian based distros also have `/etc/cron.d`, which then has hourly, daily, etc. subdivisions. You only need to add a script into one of these folders and it will be executed (if it does not have an extensions and is executable).

So you could just softlink /concourse/scripts/prune_workers.sh into the right folder and always execute that, no need to check:
```shell
ln -nfs /concourse/scripts/prune_workers.sh /etc/cron.weekly/prune_workers
```